tests/next_tests/integration_tests/feature_tests/ffront_tests/test_scalar_if.py

```diff
@@ -56,6 +56,7 @@ def simple_if(a: cases.IField, b: cases.IField, condition: bool) -> cases.IField
     cases.verify(cartesian_case, simple_if, a, b, condition, out=out, ref=a if condition else b)
 
 
+# TODO: test with fields on different domains
```

Comment:
want to add one?

Reply:
Nope :-) I'll look at the if statements when I'll clean up the ternary workarounds in PMAP and Icon4Py microphysics.